parsl/executors/high_throughput/manager_selector.py

```diff
@@ -19,7 +19,35 @@ def sort_managers(self, ready_managers: Dict[bytes, ManagerRecord], manager_list
 
 class RandomManagerSelector(ManagerSelector):
 
+    """Returns a shuffled list of interesting_managers
+
+    By default this strategy is used by the interchange. Works well
+    in distributing workloads equally across all availble compute
+    resources. Is not effective in conjunction with elastic scaling
+    behavior as it leads to wasted resource consumption.
+    """
```

Comment:
maybe put subjects in these sentences such as `it` - this isn't charged by the word.